Dockerfile

```
@@ -20,13 +15,25 @@ RUN apt-get update && apt-get install -y \
+# Set up application directory
 RUN mkdir /app
 WORKDIR /app
 
-CMD ["tail", "-f", "/dev/null"]
+COPY fitnessllm_dataplatform ./fitnessllm_dataplatform
+COPY cloud_functions ./cloud_functions
+COPY pyproject.toml poetry.lock ./
+
+# Install dependencies
+RUN poetry config virtualenvs.in-project true
+RUN poetry lock
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)